interface/usergroup/user_admin.php

```diff
@@ -559,7 +565,56 @@ function authorized_clicked() {
   </select></td>
   <td><span class=text><?php echo xlt('Additional Info'); ?>:</span></td>
   <td><textarea style="width:150px;" name="comments" wrap=auto rows=4 cols=25><?php echo $iter["info"];?></textarea></td>
+  </tr>
+  <tr>
+  <td><span class="text"><?php echo xlt('Full-screen role:'); ?></span></td>
+  <td>
+  <select style="width:120px;" name="role_name">
+    <?php
+      $fres = sqlStatement("select distinct menu_set from menu_trees order by menu_set");
+      if ($fres) {
+        for ($iter3 = 0;$frow = sqlFetchArray($fres);$iter3++)
+          $result[$iter3] = $frow;
+        foreach($result as $iter3) {
+      ?>
+      <option value="<?php echo $iter3{'menu_set'};?>" <?php if ($iter3{'menu_set'} == $iter["fullscreen_role"]) echo "selected"; ?>><?php echo $iter3{'menu_set'}; ?></option>
+      <?php
+        }
+      }
+    ?>
+  </select>
+  </td>
+  <td><span class="text"> <?php echo xlt('Full-screen page:'); ?></span></td>
+  <td>
+      <select style="width:120px;" name="fullscreen_page" id="fullscreen_page">
+      <?php
+
+      $fres = sqlStatement("select entry_id from menu_trees where menu_set='".$iter["fullscreen_role"]."'");
+      if($fres) {
+        for($iter3 = 0;$frow = sqlFetchArray($fres); $iter3++)
+          $result[$iter3] = $frow;
+          // needs to be worked on for different types of menus
+          foreach($result as $iter2) {
+            $fres2 = sqlStatement("select id,label from menu_entries where id='".$iter2{'entry_id'}."'");
+            $frow2 = sqlFetchArray($fres2);
+            ?> 
+            <option value="<?php echo $frow2{'id'}; ?>" <?php if ($frow2{'id'} == $iter["fullscreen_page"]) echo "selected"; ?>><?php echo xlt($frow2{'label'}); ?></option>
```

Comment:
add this to $iter['fulscreen_page']  attr($iter['fulscreen_page']) this will escape the field